privaterelay/pending_locales/en/pending.ftl

```diff
@@ -4,6 +4,23 @@
 
 # This is the Django equivalent of frontend/pendingTranslations.ftl
 
+# $sender (string) - the sender's email address
```

Comment:
Just a reminder to mark the removed strings as `# Deprecated` when submitting these to the l10n repo :)